Gemfile

```diff
@@ -17,9 +17,9 @@ gem 'pg'
 gem 'pundit'
 gem 'valid_email'
 gem 'rack-attack'
-gem 'ruby-saml', github: 'amoose/ruby-saml'
+gem 'ruby-saml'
 gem 'nokogiri-xmlsec-me-harder', '~> 0.9.1', require: 'xmlsec'
-gem 'saml_idp', github: '18F/saml_idp'
+gem 'saml_idp', github: 'pkarman/saml_idp', branch: 'logout-request-builder'
 gem 'sass-rails', '~> 5.0'
```

Comment:
What bundler-audit (via Code Climate) is complaining about here is that the `github` shortcut is being used over the preferred `git` directive with full `https` url, like this:
```
gem 'saml_idp', git: 'https://github.com/pkarman/saml_idp', branch: 'logout-request-builder'
```